app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1950,4 +1961,8 @@ private void searchFiles() {
       fileSearch.scan(sharedPreferenceUtil.getPrefStorage());
     }
   }
+
+  public boolean checkNull(View view) {
+    return view != null;
+  }
```

Comment:
In my opinion, encapsulating such small functionality in a separate function is overkill. You are anyways going to write `if(checkNull(view_name)){}`. Why not just `if(view_name != null)`. Every function call has an overhead and hence I am not in favor of it. If readability is your main intention, then we might leave it like you have done but I find explicit check equally readable.